src/message/MessageList.js

```diff
@@ -80,6 +82,7 @@ export default class MessageList extends React.PureComponent {
         <LoadingIndicator active={fetching.older} caughtUp={caughtUp.older} />
         {messageList}
         {!singleFetchProgress && fetching.newer && <LoadingIndicator active />}
+        {typingUsers && <MessageTyping users={typingUsers} />}
```

Comment:
Nice catch. A good argument for doing a `connect` via redux, but again, maybe later.